antarest/study/storage/abstract_storage_service.py

```diff
@@ -65,10 +65,12 @@ def __init__(
     def get_study_information(
         self,
         study: Study,
+        folder_path: Optional[str] = None,
     ) -> StudyMetadataDTO:
         additional_data = study.additional_data or StudyAdditionalData()
         study_workspace = getattr(study, "workspace", DEFAULT_WORKSPACE_NAME)
-        folder: Optional[str] = None
+        folder: Optional[str] = folder_path
+
         if hasattr(study, "folder"):
             folder = study.folder
```

Comment:
Here we override the folder passed in argument with the one already present in the study --> we return to the API the old folder name instead of the new one computed from the database.

We probably miss some test for this!